force-app/main/default/lwc/usefullLinks/__tests__/usefullLinks.test.js

```diff
@@ -0,0 +1,31 @@
+import { createElement } from 'lwc';
+import UsefullLinks from 'c/usefullLinks';
+
+describe('c-usefull-links', () => {
+    afterEach(() => {
+        // The jsdom instance is shared across test cases in a single file so reset the DOM
+        while (document.body.firstChild) {
+            document.body.removeChild(document.body.firstChild);
+        }
+    });
+
+    it('test of the url links', () => {
+        // Arrange
+        const element = createElement('c-usefull-links', {
+            is: UsefullLinks
+        });
+
+        const fullLinks = [
+            { label: 'firstLink', url: 'https://www.google.com' },
+            { label: 'SecondLink', url: 'https://www.google.com' },
```

Comment:
this does not makes sense at all. fullLinks is not an api property it does not need to be set